Formula/creduce.rb

```diff
@@ -1,9 +1,9 @@
 class Creduce < Formula
   desc "Reduce a C/C++ program while keeping a property of interest"
   homepage "https://embed.cs.utah.edu/creduce/"
-  url "https://github.com/csmith-project/creduce/archive/creduce-2.3.0.tar.gz"
-  sha256 "47a42751aab8b51bc10d8df62f359bdc1b4a644f16feb85b9f7325f0c5bce4a3"
-  revision 2
+  url "https://github.com/csmith-project/creduce/archive/creduce-2.5.0.tar.gz"
+  mirror "http://embed.cs.utah.edu/creduce/creduce-2.5.0.tar.gz"
```

Comment:
Thanks for investigating this! This makes sense and I guess per the explanation we should be exclusively using `utah.edu` as the source.